File: bindings/java/src/main/java/org/indunet/robot/bus/NativePreload.java
package org.indunet.robot.bus;

/**
 * Tell the JNA loader that {@code librobot_bus_c} is already in the process (e.g. after Android
 * {@code System.loadLibrary}). Prefer {@code RobotBusAndroid.init} on Android.
 */
public final class NativePreload {
    private NativePreload() {}

    public static void markRobotBusNativePreloaded() {
        NativeLoader.markPreloaded();
    }
}

File: bindings/kotlin/src/main/kotlin/org/indunet/robot/bus/NativePreload.kt
package org.indunet.robot.bus

/**
 * Tell the JNA loader that `librobot_bus_c` is already in the process
 * (e.g. after Android `System.loadLibrary`). Prefer `RobotBusAndroid.init` on Android.
 */
fun markRobotBusNativePreloaded() {
    NativeLoader.markPreloaded()
}
